src/pages/Debug/Report/DebugReportActions.tsx

```diff
@@ -48,11 +49,9 @@ function DebugReportActions({reportID}: DebugReportActionsProps) {
                 onPress={() => Navigation.navigate(ROUTES.DEBUG_REPORT_ACTION_CREATE.getRoute(reportID))}
                 style={[styles.pb5, styles.ph3]}
             />
-            <FlatList
-                data={sortedAllReportActions}
-                renderItem={renderItem}
-                scrollEnabled={false}
-            />
+            {/* This list was previously rendered as a FlatList, but it turned out that it caused the component to flash in some cases,
+            so it was replaced by this solution. */}
+            {sortedAllReportActions?.map((item, index) => renderItem(item, index))}
```

Comment:
Switching to `.map()` isn't ideal since `FlatList` provides built-in virtualization and better performance for large lists.

The flashing issue you mentioned might be due to the missing `keyExtractor` prop in the `FlatList`.

I suggest re-adding `FlatList` with a `keyExtractor` like this to see if it resolves the issue:

```javascript
<FlatList
    data={sortedAllReportActions}
    renderItem={renderItem}
    keyExtractor={(item) => item.reportActionID.toString()}
    scrollEnabled={false}
/>
```

Reply:
I tried it before (forgot to mention it in the proposal) and unfortunately this doesn't fix it.

Here is a demo:


https://github.com/user-attachments/assets/1daa7a08-d671-4b44-9abf-b9b037187d77